### DISCUSSION TITLE
why both Lexilla and LPeg lexers?

### DISCUSSION THREAD

**Discussion by d3m3t3r** [orbitalquark/textadept#138] [Q&A]
I don't quite understand why *both* Lexilla and LPeg are needed?  Lexilla has lexers written in C and LPeg has Lua lexers and many of them overlap.

**Comment by orbitalquark** [comment]
Textadept does not utilize Lexilla's C++ lexers. It only makes use of Lexilla's lexer library in conjunction with LPeg to provide syntax highlighting via Lua lexers. Does that make more sense?
